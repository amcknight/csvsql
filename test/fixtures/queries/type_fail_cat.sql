SELECT age
FROM test/fixtures/tables/people_camel
WHERE "Hello #" ++ 3
