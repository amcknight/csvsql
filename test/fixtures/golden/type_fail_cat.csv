Type Error in right argument of Cat (LitS "Hello #") (LitI 3)
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Expected String but found Integer